projects/ngrx.io/src/app/app.module.ts

```diff
@@ -127,6 +129,16 @@ export const svgIconProviders = [
         ServiceWorkerModule.register('/ngsw-worker.js', {
             enabled: environment.production,
         }),
+        provideFirebaseApp(() => initializeApp({
+            apiKey: 'AIzaSyBmTz0vGq-QIadpp1piHs7nA_ID8uRAxGI',
```

Comment:
This config should be in the environments files

Reply:
Do we have a dev FIrebase project?

Reply:
I was referring to the src/environments/environment.ts and related environment files.

Reply:
Yes, I understand.
Typically I'd put there something that's unique for each different environment. We don't have a separate dev Firebase project, so I'd typically leave the config where it is used.